brodybits/create-react-native-module
# support React Native 0.61; drop React Native 0.59

Major (breaking) updates for new 0.x release:

* generate example with React Native 0.61 by default
* remove `react-native link` step (not needed since React Native 0.60)
* prep: test with generate example for Android only
  _(needed to ensure `pod install` is not run on example in case the module is for Android only)_
* do `pod install` in generated example for iOS
* remove `pod install` step from manual post-create instructions
* update generated Android view for React Native 0.60(+)
* update template `devDependencies` for React Native 0.61
* minimum React Native 0.60 in template `peerDependencies`
* documentation updates
* update documentation for out-of-tree platforms
  (not supported by the generated example)

__Tested as follows:__

- [x] _library module with generated example working on RN 0.61 on Android & iOS_
- [x] _view module with generated example working on RN 0.61 on Android & iOS_
- [x] _library module with generated example working on RN 0.60 on Android & iOS_
- [x] _view module with generated example working on RN 0.60 on Android & iOS_

resolves #29
resolves #93
resolves #99
resolves #132
closes #130
closes #192
